templates/components/engines/python/agent/__init__.py

```diff
@@ -11,11 +11,12 @@ def get_chat_engine():
     top_k = os.getenv("TOP_K", "3")
     tools = []
 
-    # Add query tool
+    # Add query tool if index exists
```

Comment:
Does this mode work if the user does not have an index and no tools config as well?

Reply:
yes, tested this